### PR TITLE
Added Metric System for Weather Cog

### PR DESCRIPTION
**Pull request by Emimendoza**
Before:
image
After:
image

### CODE REVIEW

**Comment by Emimendoza** [comment]
#287

File: bot/cogs/weather_cog.py [hunk 1]
```diff
@@ -67,14 +67,14 @@ def getPageData(self, Lat, Lon, res_weather_json, city, is_cond, is_hr, is_day):
...
             # Building the Page
             page += f'Location: {city} ({Lat},{Lon})\n'
-            #page += f'Temperature: {round(temp,1)}°F\n'
-            page += f'Temperature: {temp}°F\n'
+            # page += f'Temperature: {round(temp,1)}°F\n'
+            page += f'Temperature: {temp}°F / {round((temp - 32) * (5 / 9), 2)}°C\n'
             page += f'Condition:\t{desc}\n\n'
 
-            page += f'Feels Like: {round(feels,1)}°F\n'
+            page += f'Feels Like: {round(feels, 1)}°F / {round((feels - 32) * (5 / 9), 2)}°C\n'
```
Comment (Exper1mental) [inline bot/cogs/weather_cog.py:74]: Is there a reason behind rounding Fahrenheit to one decimal but Celcius to two?

Comment (Emimendoza) [reply bot/cogs/weather_cog.py]: Yeah thats a typo, fixed it

**Review by Jay-Madden — APPROVED** [review]
lgtm

**Pull request merged**